db/migrate/20190820095253_copy_enrichments_to_next_cycle.rb

```diff
@@ -0,0 +1,20 @@
+class CopyEnrichmentsToNextCycle < ActiveRecord::Migration[5.2]
+  def up
+    current_providers = RecruitmentCycle.current_recruitment_cycle.providers.includes(:enrichments).all
+    next_providers = RecruitmentCycle.next_recruitment_cycle.providers.all
+
+    current_providers.each do |p|
+      next if p.enrichments.empty?
+
+      last_enrichment = p.enrichments.order(created_at: :desc, id: :desc).first.dup
```

Comment:
`last_enrichment` should this only be the ones that have `status` as `published`? @fofr

Reply:
I took the same logic used within copying courses to providers, which takes the latest enrichment regardless of status:

```ruby
      if (last_enrichment = enrichments.latest_first.first)
        last_enrichment.copy_to_course(new_course)
      end
```